lib/cryptoexchange/exchanges/liqui/services/market.rb

```diff
@@ -0,0 +1,43 @@
+require 'bigdecimal'
+
+module Cryptoexchange::Exchanges
+  module Liqui
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          market = "#{market_pair.base}_#{market_pair.target}".downcase
+          "#{Cryptoexchange::Exchanges::Liqui::Market::API_URL}/ticker/#{market}"
+        end
+
+        def adapt(output, market_pair)
+          market = output["#{market_pair.base}_#{market_pair.target}".downcase]
+
+          ticker = Liqui::Models::Ticker.new
+          ticker.base = market_pair.base
+          ticker.target = market_pair.target
+          ticker.market = Liqui::Market::NAME
+          ticker.last = BigDecimal.new(market['last'].to_s)
```

Comment:
@yihangho I realised that for some exchanges it can be `nil` with the key and that will crash. Can you refer to my `BigDecimal` PR?